### PR TITLE
DED-520 Update Supervisor app README

### PR DESCRIPTION
**Pull request by kamilatokarska**
When it comes to the README itself, I took it from the existing description [here](https://github.com/livechat/sample-apps/tree/master/supervisor). I added a section mentioning two remaining sample apps. I decided to leave the last section as it is, meaning talking about livechat, as I am not sure if we can introduce text.com already. If I can already make it as Text, then please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,69 @@
...
+Experiment more with two different sample apps:
+ - [Tag Master](https://github.com/livechat/tag-master) - widget for helping your agents manage tags and canned responses.
+ - [Progress](https://github.com/livechat/progress-app) - widget for helping you monitor statistics of your team, such as chat ratings, response times, and chatting times.
+
+# Feedback
+If you find some bugs, please create an issue in this repo. We will try to fix is ASAP ;)
+
+# If you're new to LiveChat
 
 **LiveChat** is an online customer service software with live support, help desk software, and web analytics capabilities. It's used by more than 27,000 companies all over the world. For more info, check out [LiveChat for Developers](https://developers.livechatinc.com/).
```
Comment (kamilatokarska) [inline README.md:65]: now noticed that the number of customers is wrong

File: README.md [hunk 2]
```diff
@@ -1,20 +1,69 @@
...
+    * Run your app (`npm start`).
+8. Add your app url (for example: `https://localhost:3000`) in these two locations:
+
+    * **Redirect URI whitelist**
+    * **Agent App Widgets**
+9. In **Private installation**, click **Install app**. 
+
+You should now be able to use **Supervisor** with LiveChat.
+
+# How it works
```
Comment (ale-kacperczyk) [inline README.md:45]: There are some old links in this part

File: README.md (hunk 1 above)
Comment (kamilatokarska) [inline README.md:65]:
```suggestion
**LiveChat** is an online customer service software with live support, help desk software, and web analytics capabilities. It's used by more than 36,000 companies all over the world. For more info, check out [LiveChat for Developers](https://developers.livechatinc.com/).
```

Comment (kamilatokarska) [inline README.md:65]:
```suggestion
**LiveChat** is an online customer service software with live support, help desk software, and web analytics capabilities. It's used by more than 37,000 companies all over the world. For more info, check out [LiveChat for Developers](https://developers.livechatinc.com/).
```

**Comment by kamilatokarska** [comment]
I will create a new PR with the new README, as it would be easier for me to introduce all the changes in the edit mode of the README. I noticed that the descriptions of all the 3 apps have the same or similar mistakes to the ones you mentioned, so when we approve this README, I'll correct the descriptions of the remaining two apps as well.

**Pull request closed without merging**